src/main/java/com/dynatrace/openkit/protocol/Beacon.java

```diff
@@ -783,4 +786,12 @@ private long getTimeSinceSessionStartTime(long timestamp) {
     public boolean isEmpty() {
         return beaconCache.isEmpty(sessionNumber);
     }
+
+    /**
```

Comment:
/**
  Returns the session number
  @return the session number
*/

sorry about the missing star characters, mardown would start a list here